File: src/ClientApp/src/app/pages/clients/view-models/client-contact.view-model.ts
export interface ClientContactViewModel {
    id: string;
    clientId: string;
    firstName: string;
    lastName: string;
    fullName: string;
    designation?: string | null;
    department?: string | null;
    email?: string | null;
    phone?: string | null;
    mobile?: string | null;
    isPrimary: boolean;
    status: number;
    statusName: string;
    notes?: string | null;
    isActive: boolean;
    createdAt: string;
    updatedAt?: string | null;
}

File: src/API/src/Modules/Clients/Clients.Application/ViewModels/ClientContactViewModel.cs
using Clients.Domain.Enums;

namespace Clients.Application.ViewModels
{
    public class ClientContactViewModel
    {
        public Guid Id { get; set; }
        public Guid ClientId { get; set; }
        public string FirstName { get; set; } = string.Empty;
        public string LastName { get; set; } = string.Empty;
        public string FullName { get; set; } = string.Empty;
        public string? Designation { get; set; }
        public string? Department { get; set; }
        public string? Email { get; set; }
        public string? Phone { get; set; }
        public string? Mobile { get; set; }
        public bool IsPrimary { get; set; }
        public ClientContactStatus Status { get; set; }
        public string StatusName { get; set; } = string.Empty;
        public string? Notes { get; set; }
        public bool IsActive { get; set; }
        public DateTime CreatedAt { get; set; }
        public DateTime? UpdatedAt { get; set; }
    }
}
